magento/magento2
# ISSUE-38593 - Fix to save middle name in the quote for guest

### Description (*)
Fix to save the customer's middle name in the quote/order table for the guest checkout

### Related Pull Requests
NA

### Fixed Issues (if relevant)
1. Fixes magento/magento2#38593

### Manual testing scenarios (*)
1. Place guest order with middle name.
2. Verify the quote table whether the "customer_middlename" column is filled or not.

### Questions or comments
N/A

### Contribution checklist (*)
